### ISSUE TITLE
About the color of sysmon under white theme.

### ISSUE THREAD

**Issue by elegantflamingo22** [kingToolbox/WindTerm#1350]
![image](https://user-images.githubusercontent.com/110973278/201652060-c6b857be-a910-4fba-abfd-0d8f53ab7f05.png)

As shown in the picture, this color cannot be seen normally under the white theme.

**Comment by funkpopo** [comment]
You can check the theme file in the following path to find out whether there are related color setting options. The default white theme name is "dige-white".

**Path:\WindTerm\global\themes\**

**Comment by elegantflamingo22** [comment]
Yes, maybe what I described is not clear enough, I found the path to adjust the theme, but I didn't find an option to adjust sysmon'color, because I prefer to use a light theme, but this will cause Sysmon'view to have a very low contrast, not as good as a black theme see clearly,as shown in the pictureAs shown in the picture
![image](https://user-images.githubusercontent.com/110973278/201913754-dfbbc2a3-04a3-4a03-9047-3ee3e9b9f7ff.png)
![image](https://user-images.githubusercontent.com/110973278/201913795-5e3e12b0-0436-4a14-9bce-0f71f55c7194.png)
